src/sentry/templates/sentry/emails/relocation_started.html

```diff
@@ -0,0 +1,16 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+    <h3>Relocation Accepted</h3>
+    <p>Your relocation (id: <i>{{ uuid }}</i>) request has been accepted. You requested that the following organizations be migrating to sentry.io:</p>
```

Comment:
It feels a bit weird to have the uuid in parentheses here, would it be better to have that in smaller text underneath the header `Relocation Accepted`? Not a designer though, so maybe I don't know what I'm talking about

Reply:
Done.

I've updated the images attached to the PR to reflect this.